test/drenv/minikube.py

```diff
@@ -104,6 +105,139 @@ def ssh(profile, command):
     _watch("ssh", command, profile=profile)
 
 
+def setup_files():
+    """
+    Set up minikube to work with drenv. Must be called before starting the
+    first cluster.
+
+    To load the configuration you must call load_files() after a cluster is
+    created.
+    """
+    _setup_sysctl()
+    _setup_systemd_resolved()
+
+
+def load_files(profile):
+    """
+    Load configuration done in setup_files() before the minikube cluster was
+    started.
+
+    Must be called after the cluster is started, before running any addon. Not
+    need when starting a stopped cluster.
+    """
+    _load_sysctl(profile)
+    _load_systemd_resolved(profile)
+
+
+def cleanup_files():
+    """
+    Cleanup files added by setup_files().
+    """
+    _cleanup_file(_systemd_resolved_drenv_conf())
+    _cleanup_file(_sysctl_drenv_conf())
+
+
+def _setup_sysctl():
+    """
+    Increase fs.inotify limits to avoid random timeouts when starting kubevirt
+    VM.
+
+    We use the same configuration as OpenShift worker node.
+    See also https://www.suse.com/support/kb/doc/?id=000020048
+    """
+    path = _sysctl_drenv_conf()
+    data = """# Added by drenv setup
+fs.inotify.max_user_instances = 8192
+fs.inotify.max_user_watches = 65536
+"""
+    logging.debug("[minikube] Writing drenv sysctl configuration %s", path)
+    _write_file(path, data)
+
+
+def _load_sysctl(profile):
+    if not os.path.exists(_sysctl_drenv_conf()):
+        return
+    logging.debug("[%s] Loading drenv sysctl configuration", profile)
+    ssh(profile, "sudo sysctl -p /etc/sysctl.d/99-drenv.conf")
+
+
+def _sysctl_drenv_conf():
+    return _minikube_file("etc", "sysctl.d", "99-drenv.conf")
+
+
+def _setup_systemd_resolved():
+    """
+    Disable DNSSEC in systemd-resolved configuration.
+
+    This is workaround for minikube regression in 1.33.0:
+    https://github.com/kubernetes/minikube/issues/18705
+
+    TODO: Remove when issue is fixed in minikube.
+    """
+    path = _systemd_resolved_drenv_conf()
+    data = """# Added by drenv setup
+[Resolve]
+DNSSEC=no
+"""
```

Comment:
Minikube is adopting this workaround, so we can probably remove this very soon:
https://github.com/kubernetes/minikube/pull/18830